ntoskrnl/ntos.cmake

```diff
@@ -232,6 +232,7 @@ list(APPEND SOURCE
     ${REACTOS_SOURCE_DIR}/ntoskrnl/mm/ARM3/zeropage.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/mm/balance.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/mm/freelist.c
+    ${REACTOS_SOURCE_DIR}/ntoskrnl/mm/kasan.c
```

Comment:
This file should be added to compilation conditionally, only if `KASAN_ENABLED` is defined.